velox/dwio/dwrf/common/CacheInputStream.cpp

```diff
@@ -157,6 +158,65 @@ void CacheInputStream::loadSync(dwio::common::Region region) {
     if (entry->isExclusive()) {
       entry->setGroupId(groupId_);
       entry->setTrackingId(trackingId_);
+      auto ssdCache = cache_->ssdCache();
+
+      if (ssdCache) {
+        auto& file = ssdCache->file(fileNum_);
+        auto ssdPin =
+            file.find(cache::RawFileCacheKey{fileNum_, region.offset});
+        if (!ssdPin.empty() && ssdPin.run().size() < entry->size()) {
+          LOG(INFO) << fmt::format(
+              "IOERR: Ssd entry for {}:{} {} b shorter than requested {}b",
```

Comment:
This message seems off. It says "entry N b shorter than requested Mb". I assume "b" is for bytes. It reads as entry is N bytes shorter than M, but N is not a difference in bytes, rather is it the size of the entry. It would be nice to reword this message for clarity.

Also, why is this an OK condition? Why is logging enough?